warrior/Framework/ClassUtils/WNetwork/warrior_cli_class.py

```diff
@@ -31,6 +31,7 @@
 from Framework.Utils.data_Utils import get_object_from_datarepository
 from WarriorCore.Classes.warmock_class import mocked
 from timeit import itertools
+from Framework.Utils.config_Utils import data_repository
```

Comment:
I am unsure what resp_pat_key has to do with keeping the data_response in the data_repository. Can you please explain the changes made in this pull request?

Reply:
Hi @sanikakulkarni,

Attached the mail thread in Jira ticket and forwarded the same mail thread to you as well.
This commit was fixed by Reshma and user already tested it and it is working fine.

Reply:
Hi @apathapa-warrior,

I have the email with me and I understand the request that generated this ticket. I am still unsure as to how the resp_pat_key relates to the changes requested in the email. Can you please explain the changes made here and how to test those changes?

Thanks,
Sanika